### PR TITLE
DOC-813: Document fixed_toolbar_container_target setting added in TinyMCE 5.8

### PR DESCRIPTION
**Pull request by ashfordneil**
Related Ticket: DOC-813

Description of Changes:
* Add `fixed_toolbar_container_target` setting to the configuration/editor-appearance section
* Added a release note linking to it, and attributing @pvrobays

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes58.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ The following new features were added for the {{site.productname}} 5.8 release.
 
 ### Feature name
 
+### Added new `fixed_toolbar_container_target` setting
+
+Added a new `fixed_toolbar_container_target` setting which renders the toolbar in the specified `HTMLElement`. Patch contributed by [pvrobays](https://github.com/pvrobays).
+
+For more information on `fixed_toolbar_container_target`, see [User interface options: `fixed_toolbar_container_target`]({{ site.baseurl }}/configure/editor-appearance/#fixed_toolbar_container_target).
```
Comment (lnewson) [inline release-notes/release-notes58.md:34]: "More" is generally discouraged based on previous PRs... we also have template for this if you look at the older release notes, so it should be something like this instead:
```suggestion
For information on the `fixed_toolbar_container_target` option, see [User interface options - `fixed_toolbar_container_target`]({{ site.baseurl }}/configure/editor-appearance/#fixed_toolbar_container_target).
```

**Review by lnewson — CHANGES_REQUESTED** [review]

File: release-notes/release-notes58.md [hunk 2]
```diff
@@ -27,6 +27,12 @@ The following new features were added for the {{site.productname}} 5.8 release.
 
 ### Feature name
 
+### Added new `fixed_toolbar_container_target` setting
+
+Added a new `fixed_toolbar_container_target` setting which renders the toolbar in the specified `HTMLElement`. Patch contributed by [pvrobays](https://github.com/pvrobays).
```
[the remark this answers is not in the document]
Comment (ashfordneil) [reply release-notes/release-notes58.md:32]: The first sentence sounds like you're changing it to use passive voice, so I'd rather find a different way to fix that if you don't like the original. Happy to take the second sentence though, it sounds a lot nicer

[the remark this answers is not in the document]
Comment (lnewson) [reply release-notes/release-notes58.md:32]: It definitely needs changing and should not start with `Added`. How you do that is up to you, but remember it's meant to be more like a marketing spiel to say what's been added and what it does.

Comment (JamesToohey) [inline release-notes/release-notes58.md:32]: How about something like

```suggestion
The new `fixed_toolbar_container_target` option allows the toolbar to be rendered in a specified `HTMLElement`. This feature was contributed by [pvrobays](https://github.com/pvrobays).
```

Comment (lnewson) [reply release-notes/release-notes58.md:32]: Yeah, this looks good to me and matches what's normally used for release notes. We may also want to mention it's similar to the `fixed_toolbar_container` option here just so people are aware when reading this without going to have to read the full details in the configuration docs.

File: _includes/configuration/fixed-toolbar-container-target.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+## `fixed_toolbar_container_target`
+
+{{ site.requires_5_8v }}
+
+Use this option to render the inline toolbar into a fixed positioned HTML element, similarly to [`fixed_toolbar_container`]({{ site.baseurl }}/configure/editor-appearance/#fixed_toolbar_container). The difference between these two options is that `fixed_toolbar_container_target` accepts a `HTMLElement` directly, instead of a CSS selector.
```
Comment (tylerkelly13) [inline _includes/configuration/fixed-toolbar-container-target.md:5]:
```suggestion
Use this option to render the inline toolbar into a fixed-positioned HTML element by passing a HTMLElement` directly to the option. This option is similar to the [`fixed_toolbar_container` option]({{ site.baseurl }}/configure/editor-appearance/#fixed_toolbar_container), which accepts a CSS selector.
```
Don't "bury the lead" :wink:

**Review by tylerkelly13 — CHANGES_REQUESTED** [review]

**Comment by tylerkelly13** [comment]
just a couple more changes please.

**Review by lnewson — APPROVED** [review]
Looks fine to me (taking into account Tyler's comments). Thanks! :+1:

File: _includes/configuration/fixed-toolbar-container-target.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+## `fixed_toolbar_container_target`
+
+{{ site.requires_5_8v }}
+
+Use this option to render the inline toolbar into a fixed-positioned HTML element by passing a HTMLElement` directly to the option. This option is similar to the [`fixed_toolbar_container` option]({{ site.baseurl }}/configure/editor-appearance/#fixed_toolbar_container), which accepts a CSS selector.
```
Comment (tylerkelly13) [inline _includes/configuration/fixed-toolbar-container-target.md:5]:
```suggestion
Use this option to render the inline toolbar into a fixed-positioned HTML element by passing a `HTMLElement` directly to the option. This option is similar to the [`fixed_toolbar_container` option]({{ site.baseurl }}/configure/editor-appearance/#fixed_toolbar_container), which accepts a CSS selector.
```

**Review by tylerkelly13 — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**